src/app/components/BookFilters/BookFilters.jsx

```diff
@@ -1,38 +1,84 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FilterIcon } from 'dgx-svg-icons';
+import {
+  contains as _contains,
+  findWhere as _findWhere,
+} from 'underscore';
 
 import Filter from './Filter';
-import Actions from '../../actions/BookActions';
 
 class BookFilters extends React.Component {
   constructor(props) {
     super(props);
 
+    this.state = {
+      // Create an array data structure of filter objects.
+      filters: this.props.filters.map(filter => ({
+        active: false,
+        id: filter.toLowerCase().split(' ').join('-'),
+        label: filter,
+      })),
+      focusId: '',
+    };
+
     this.renderItems = this.renderItems.bind(this);
     this.onClick = this.onClick.bind(this);
+    this.getFilterArray = this.getFilterArray.bind(this);
   }
 
   onClick(filterId, active) {
-    Actions.setSelectedFilter(filterId, active);
+    // const filterId = filter.toLowerCase().split(' ').join('-');
+    const f = _findWhere(this.state.filters, { id: filterId });
```

Comment:
I would promote better naming convention instead of `f` -- following our engineering principles. See https://github.com/NYPL/engineering-general/blob/master/standards/naming-conventions.md#identifiers

Reply:
Updated. Thanks.